lib/compat/wordpress-6.1/blocks.php

```diff
@@ -5,6 +5,27 @@
  * @package gutenberg
  */
 
+/**
+ * Update allowed inline style attributes list.
+ *
+ * Note: This should be removed when the minimum required WP version is >= 6.1.
+ *
+ * @param string[] $attrs Array of allowed CSS attributes.
+ * @return string[] CSS attributes.
+ */
+function gutenberg_safe_style_attrs_6_1( $attrs ) {
```

Comment:
To see if there are any blockers with adding in support for these properties, I've opened up a Trac ticket here: https://core.trac.wordpress.org/ticket/56122#ticket